specification/compute/resource-manager/Microsoft.Compute/DiskRP/stable/2022-07-02/examples/snapshotExamples/Snapshot_Get.json

```diff
@@ -45,6 +45,7 @@
               }
             ]
           },
+          "incrementalSnapshotFamilyId": "5382cd9f-c419-4984-9fa7-4825e07f9fe9",
```

Comment:
This property is only present if incremental property is true. Do we have an example for Get on Incremental snapshot?

Reply:
Added a new example for incremental snapshot